sched/jobDefSerializer.go

```diff
@@ -0,0 +1,113 @@
+package sched
```

Comment:
Sorry for missing this initially, I think we're trying to prefer foo_bar.go to fooBar.go